cmd/massastation/main.go

```diff
@@ -59,18 +59,19 @@ func ParseFlags() api.StartServerFlags {
 }
 
 func main() {
+	err := initialize.Logger()
+	if err != nil {
+		log.Fatalf("while initializing logger: %s", err.Error())
+	}
+
```

Comment:
defer logger.Close() should be here

Reply:
yeah I had lint issue because bellow we do `os.Exit(0)`: "exitAfterDefer: os.Exit will exit, and `defer logger.Close()` will not run"

*Side note:*
Why I move the logger initialization: because `ParseFlags` use the logger.